### PR TITLE
feat/softsign: Add softsign function

### PR DESCRIPTION
**Pull request by ojetokun**
Added softsign activation function
## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Added softsign activation function
Issue Number: N/A

## What is the new behavior?

`NN::softsign(@tensor)`

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by raphaelDkhn** [comment]
@ojetokun can you run please `scarb run docgen` to generate documentation from your docstrings ?

File: docs/apis/operators/tensor/tensor.new.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# tensor.new
+# TensorTrait::new
```
Comment (franalgaba) [inline docs/apis/operators/tensor/tensor.new.md:1]:
```suggestion
# tensor.new
```

File: docs/apis/numbers/fixed-point/README.md [hunk 2]
```diff
@@ -35,23 +35,23 @@ use orion::numbers::fixed_point::Fixed;
...
-| [`fp.ln`](fp.ln.md)                                   | Returns the natural logarithm of the fixed point number.                                        |
-| [`fp.log2`](fp.log2.md)                               | Returns the base-2 logarithm of the fixed point number.                                         |
-| [`fp.log10`](fp.log10.md)                             | Returns the base-10 logarithm of the fixed point number.                                        |
-| [`fp.pow`](fp.pow.md)                                 | Returns the result of raising the fixed point number to the power of another fixed point number |
-| [`fp.round`](fp.round.md)                             | Rounds the fixed point number to the nearest whole number.                                      |
-| [`fp.sqrt`](fp.sqrt.md)                               | Returns the square root of the fixed point number.                                              |
+| function | description |
+| --- | --- |
+| [`fp.new`](https://orion.gizatech.xyz/apis/numbers/fixed-point/fp.new.md) | Constructs a new `FixedType` instance. |
```
Comment (franalgaba) [inline docs/apis/numbers/fixed-point/README.md:40]: Double check @raphaelDkhn

File: docs/apis/compatibility.md [hunk 3]
```diff
@@ -28,4 +29,4 @@ Performance optimizations:
 | :----------------: | :----------------: |
 | 8-bit quantization | :white_check_mark: |
 
-Current Operators support: **15/156 (9%)**
+Current Operators support: **16/156 (10%)**
```
Comment (franalgaba) [inline docs/apis/compatibility.md:32]: Update if other PR is merged before this one

**Pull request merged**